client-app/pages/account/list-details.vue

```diff
@@ -305,6 +313,7 @@ async function canChangeRoute() {
 onBeforeRouteLeave(canChangeRoute);
 onBeforeRouteUpdate(canChangeRoute);
 
+// eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
As discussed in dev chat, it's better to configure [no-misused-promises](https://typescript-eslint.io/rules/no-misused-promises/) to ignore such case
```json
{
  "@typescript-eslint/no-misused-promises": [
    "warn",
    {
      "checksVoidReturn": {
        "arguments": false,
      }
    }
  ]
}
```